app/uk/gov/hmrc/securemessage/models/QueryMessageRequest.scala

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.securemessage.models
+
+import org.joda.time.DateTime
+import play.api.libs.json.{ Format, JodaReads, JodaWrites, Json, OWrites, Reads, Writes }
+
+/*
```

Comment:
can you move it to test/resources .json file and reference the link here ?

Reply:
meant to delete

Reply:
oh yes will add to test json resources